chef/cookbooks/apache2/recipes/mod_ssl.rb

```diff
@@ -40,6 +40,7 @@
   # override the resource defined in default.rb; we don't want to create the
   # resource again, otherwise we will write the file twice
   resource = resources(template: "#{node[:apache][:dir]}/ports.conf")
+  node[:apache][:listen_ports] << "443"
   resource.variables({apache_listen_ports: [node[:apache][:listen_ports], "443"].flatten})
```

Comment:
Here you are creating a list containing - as the first element - the list of listen_ports, and - as the second element - "443" and then flattening that down to a simple list, which should effectively be the listen_ports list with 443 entry appended...

Reply:
Hah, you're right, I forgot to change the code after; will fix...